articles/architecture-scenarios/b2c/planning.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Planning Your Consumer Identity and Access Management Integration
+description: Why it's important to plan your Consumer Identity and Access Management Implementation prior to beginning the integration
+toc: false
+topics:
+    - b2c
+    - ciam
+contentType: concept
+useCase:
+  - implementation
+---
+# Planning Your Consumer Identity and Access Management Integration
+
+In our experience, customers who establish clear objectives and align those objectives to requirements early on benefit the most from their integration with Auth0. Regardless of the project type, we recommend that you plan your project using a phased approach across multiple workstreams. In most cases, we see the most success with those adopting a three-phase project structure.
+
+::: note
+Identifying your primary objectives early on during the project will help you focus on the specifics important to building out your solution. For instance, if your primary objective is to avoid disruption to end users, then you should not adopt a "big bang" approach when integrating with Auth0.
```

Comment:
```suggestion
Identifying your primary objectives early on during the project will also help you focus on the specifics important to building out your solution. For instance, if your primary objective is to avoid disruption to end users, then you should not adopt a "big bang" approach when integrating with Auth0.
```